pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -77,7 +79,8 @@ stdenv.mkDerivation (finalAttrs: {
     ++ lib.optional (postgresql != null) postgresql;
 
   nativeBuildInputs = [ bison flex gperf lndir perl pkg-config which ]
-    ++ lib.optionals stdenv.isDarwin [ xcbuild ];
+    ++ lib.optionals stdenv.isDarwin [ xcbuild ]
+    ++ lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) buildPackages.stdenv.cc ];
```

Comment:
```suggestion
    ++ lib.optional (stdenv.hostPlatform != stdenv.buildPlatform) buildPackages.stdenv.cc;
```

Reply:
I have no opinion on `lib.optional` vs `lib.optionals`. Issue here is the bracket that never started. As it is a single use, I usually use `lib.optional`. If remember there was a debate once related to this but I don't know if consensus was reached.